.github/workflows/pages.yaml

```diff
@@ -31,12 +38,14 @@ jobs:
         run: uv pip install .[docs]
 
       - name: Generate HTML
-        run: uv run sphinx-build -d _build/doctrees docs/source _build/html
+        run: |
```

Comment:
I also set the `-j auto` flag here to take advantage of parallel building within Sphinx.